simtbx/diffBragg/tests/tst_diffBragg_diffuse_properties.py

```diff
@@ -16,6 +16,7 @@
 parser.add_argument("--stencil", type=int, default=0, help="sets the stencil property in diffBragg (default is 0)")
 parser.add_argument("--laue", action="store_true", help="sets the laue group number for the spacegroup")
 parser.add_argument("--grad", choices=['sigma','gamma'], default='gamma')
+parser.add_argument("--orientation", choices=[0,1], type=int, help="selects orientation for aniso. model", default=0)
```

Comment:
Could you add what option 0 and 1 mean? Something like:

"selects orientation for aniso. model (0: 45 deg to xy axes; 1: parallel to xyz axes)